modules/efk-logging-fluentd-tolerations.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-fluentd.adoc
+
+[id="efk-logging-fluentd-tolerations_{context}"]
+= Using tolerations to control the log collector pod placement
+
+You can ensure which nodes the logging collector pods run on
```

Comment:
@mburke5678 I don't think we need  taint and tolerations to controll the collection pods. @jcantrill  How do you think?

Reply:
@ewolinetz Any thoughts ^^

Reply:
IMO, collectors are supposed to run on every node.  Isn't part of taints and tolerations also to ensure the pod doesn't get evicted for things like memory and cpu pressures? If so then this would be a way to guard against those situations

Reply:
Should we specify this in the doc:
"pods that tolerate the taint without specifying tolerationSeconds in their toleration specification remain bound forever"
https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#taint-based-evictions

Reply:
@anpingli PTAL at the comments above. 
Also, you can see the conversations in Slack:
https://coreos.slack.com/archives/GGUR75P60/p1568903576072500
https://coreos.slack.com/archives/GGUR75P60/p1568903649073100